### PR TITLE
adding support for Sodaq One 3G

### PR DESCRIPTION
**Pull request by dmamalis**

### CODE REVIEW

File: examples/test_3Gbee/test_3Gbee.ino [hunk 1]
```diff
@@ -71,16 +78,18 @@ void setup()
...
     delay(500);
 
-#if defined(ARDUINO_SODAQ_AUTONOMO)
-    sodaq_3gbee.init(Serial1, BEE_VCC, BEEDTR, BEECTS);
-#elif defined(ARDUINO_SODAQ_WDT)
-    sodaq_3gbee.init_wdt(Serial1, U2_ON);
-#elif defined(ARDUINO_SODAQ_MBILI)
-    sodaq_3gbee.init(Serial1, -1, BEEDTR, BEECTS);
-#endif
+    #if defined(ARDUINO_SODAQ_AUTONOMO)
```
Comment (keestux) [inline examples/test_3Gbee/test_3Gbee.ino]: Can you undo the indenting for these lines?

**Comment by keestux** [comment]
Thanks.
Now the next request, do you know how to squash? The two commits can be squashed together and you can "forced" push your new commit to this same Pull Request.

**Comment by keestux** [comment]
I can do the squash, if you want me to do it.

**Comment by dmamalis** [comment]
Hm, i thoguht I did squash the commits. But please proceed as you wish :-)

**Comment by keestux** [comment]
Sorry, for some reason I didn't see the update. All is good.

**Pull request merged**